test/unit/io/LasWriterTest.cpp

```diff
@@ -1183,6 +1183,39 @@ TEST(LasWriterTest, pdal_add_vlr)
     EXPECT_EQ(nodes.size(), 2UL);
 }
 
+TEST(LasWriterTest, evlroffset)
+{
+    std::string outfile(Support::temppath("evlr.las"));
+
+    FileUtils::deleteFile(outfile);
+    {
+        LasReader r;
+        Options ro;
+        ro.add("filename", Support::datapath("las/wontcompress3.las"));
+        r.setOptions(ro);
+
+
+        LasWriter w;
+        Options wo;
+        std::vector<uint8_t> largeVlr(66000);
+        std::string vlr = " [ { \"description\": \"A description under 32 bytes\", \"record_id\": 42, \"user_id\": \"hobu\", \"data\": \"" +Utils::base64_encode(largeVlr)+"\" }]";
+        wo.add("vlrs", vlr);
+        wo.add("minor_version", 4);
+        wo.add("dataformat_id", 6);
+        wo.add("filename", outfile);
+        w.setOptions(wo);
+        w.setInput(r);
+
+        PointTable t;
+        w.prepare(t);
+        w.execute(t);
+        LasTester tester;
+        LasHeader *h = tester.header(w);
+        EXPECT_GT(h->eVlrOffset(),0);
```

Comment:
There's really no reason not to test the actual value.  There are a lot of wrong values that are greater than 0.

Reply:
Sure... Will add a better test.